examples/easy_start/nipype_preproc_spm_auditory.py

```diff
@@ -9,12 +9,14 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import nibabel
-from nipy.modalities.fmri.experimental_paradigm import BlockParadigm
-from nipy.modalities.fmri.design_matrix import make_dmtx
-from nipy.modalities.fmri.glm import FMRILinearModel
 from pypreprocess.nipype_preproc_spm_utils import do_subjects_preproc
 from pypreprocess.datasets import fetch_spm_auditory
 from pypreprocess.reporting.glm_reporter import generate_subject_stats_report
+from pandas import DataFrame
```

Comment:
It more common to do (like with `numpy`): `import pandas as pd` and then use `pd.Dataframe`.